src/App.css

```diff
@@ -59,6 +59,17 @@
   font-display: swap;
 }
 
+@font-face{

```

Comment:
I might be wrong, but I think the designers consolidated the fonts so that Arimo is unnecessary